internal/manager/config.go

```diff
@@ -92,6 +92,9 @@ type Config struct {
 
 	// Feature Gates
 	FeatureGates map[string]bool
+
+	// Delay Signal Handler
```

Comment:
I just realized we have a history in this area of the code of not putting proper godoc comments on these attributes. That is of course not your fault in the slightest, but since we're here, we might as well do so for this new attribute:

```suggestion
	// TermDelay is the time.Duration which the controller manager will wait
	// after receiving SIGTERM or SIGINT before shutting down. This can be
	// helpful for advanced cases with load-balancers so that the ingress
	// controller can be gracefully removed/drained from their rotation.
```